packages/apps/spaces/app/organizations/[id]/components/Timeline/TimelineActions/TimelineActions.tsx

```diff
@@ -118,31 +118,31 @@ export const TimelineActions: React.FC<TimelineActionsProps> = ({
   return (
     <Box bg='gray.25'>
       <ButtonGroup
-        // mt={6}
         position='sticky'
-        py={2}
-        border='1px dashed var(--gray-200, #EAECF0)'
-        p={2}
-        borderRadius={30}
+        border='1px solid'
+        borderColor='gray.200'
+        p='2'
+        borderRadius='full'
         bg='white'
         top='0'
-        left={6}
-        zIndex={1}
-        transform='translateY(6px)'
+        left='6'
+        zIndex='1'
+        transform='translateY(5px)'
       >
         <Button
           variant='outline'
           onClick={() => handleToggle()}
           borderRadius='3xl'
           size='xs'
-          leftIcon={<Envelope color='inherit' height={16} width={16} />}
+          leftIcon={<Icons.Mail1 boxSize='4' />}
```

Comment:
This change should not be needed 🤔  As agreed before, we need to talk about new script to generate icon components cause maintaining large objects will become gruesome

Reply:
I chose to do this as a fast solution since I already had the icon imported.